CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2.3.2 / 2018-07-12
+
+* [BUGFIX] Fix various tsdb bugs (#4369)
```

Comment:
Can we settle on one formatting for the linked PRs? It was without parens in the 2.3.1 Changelog. With parens is better, but maybe update the 2.3.1 one then too?

Reply:
Btw., these don't automatically turn into links, they have to be linked manually...

Reply:
This is what was easiest to copy&paste, if the people pushing for this change want to offer tooling for it I'll consider using it. They are auto-linked, see https://github.com/prometheus/prometheus/releases/tag/v2.3.1

Reply:
Ah, good to know it shows there. It doesn't show as links on https://github.com/prometheus/prometheus/blob/master/CHANGELOG.md

Kubernetes does real links, see https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#changelog-since-v198. But if that's too much work to do for now, we can also wait for tooling.

Reply:
I'd still at least keep the format consistent with 2.3.1 (with my pref being updating 2.3.1 to the parenthesized format).